include/seqan3/range/view/kmer_hash.hpp

```diff
@@ -61,47 +630,51 @@ struct kmer_hash_fn
-     *
-     * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)                     |
-     * |---------------------------------|:-------------------------------------:|:--------------------------------------------------:|
-     * | std::ranges::InputRange         | *required*                            | *preserved*                                        |
-     * | std::ranges::ForwardRange       | *required*                            | *preserved*                                        |
-     * | std::ranges::BidirectionalRange |                                       | *preserved*                                        |
-     * | std::ranges::RandomAccessRange  |                                       | *preserved*                                        |
-     * | std::ranges::ContiguousRange    |                                       | *lost*                                             |
-     * |                                 |                                       |                                                    |
-     * | std::ranges::ViewableRange      | *required*                            | *guaranteed*                                       |
-     * | std::ranges::View               |                                       | *guaranteed*                                       |
-     * | std::ranges::SizedRange         |                                       | *preserved*                                        |
-     * | std::ranges::CommonRange        |                                       | *preserved*                                        |
-     * | std::ranges::OutputRange        |                                       | *lost*                                             |
-     * | seqan3::ConstIterableRange      |                                       | *preserved*                                        |
-     * |                                 |                                       |                                                    |
-     * | seqan3::reference_t             | seqan3::Semialphabet                  | std::size_t                                        |
-     *
-     * See the \link view view submodule documentation \endlink for detailed descriptions of the view properties.
-     *
-     * ### Example
-     * \snippet test/snippet/range/view/kmer_hash.cpp usage
-     * \hideinitializer
-     */
-    inline auto constexpr kmer_hash = detail::kmer_hash_fn{};
+/*!\name Alphabet related views
+ * \{
+ */
+
+/*!\brief               Computes hash values for each position of a range via a given shape.
+ * \tparam urng_t       The type of the range being processed. See below for requirements. [template parameter is
+ *                      omitted in pipe notation]
+ * \param[in] urange    The range being processed. [parameter is omitted in pipe notation]
+ * \param[in] shape_    The seqan3::shape that determines how to compute the hash value.
```

Comment:
```suggestion
 * \param[in] shape     The seqan3::shape that determines how to compute the hash value.
```

This doesn't refer to the actual parameter anyway so one might as well use a nicer name,